lib/util/Consts.ts

```diff
@@ -24,6 +24,13 @@ export enum TypeURL {
   XSD_DATE_TIME_STAMP = 'http://www.w3.org/2001/XMLSchema#dateTimeStamp',
   XSD_DATE = 'http://www.w3.org/2001/XMLSchema#date',
 
+  XSD_G_MONTH = 'http://www.w3.org/2001/XMLSchema#gMonth',
+  XSD_G_MONTHDAY = 'http://www.w3.org/2001/XMLSchema#gMonthDay',
+  XSD_G_YEAR = 'http://www.w3.org/2001/XMLSchema#gYear',
+  XSD_G_YEAR_MONTH = 'http://www.w3.org/2001/XMLSchema#gYearMonth',
+  XSD_TIME = 'http://www.w3.org/2001/XMLSchema#time',
+  XSD_G_DAY = 'http://www.w3.org/2001/XMLSchema#gDay',
```

Comment:
I added these types here just because they are in the graph of supported types. We don't do anything with them, unfortunately. 

It seems like the spec is not shure what to do with them as they are not listed in: https://github.com/w3c/sparql-12/blob/main/SEP/SEP-0002/sep-0002.md
And are mentioned in a TODO question in the test files: https://github.com/kasei/sparql-12/tree/xsd_datetime_duration/tests/xsd_functions

@rubensworks would it be interesting to explicitly ask/ reunite the discussion on the github spec or do I just leave this unsupported?

Reply:
> @rubensworks would it be interesting to explicitly ask/ reunite the discussion on the github spec or do I just leave this unsupported?

Let's leave it unsupported for this PR.
But if you can think of a reasonable implementation of them, feel free to implement them in a next PR.
Based on that implementation, we can contribute support for them in the SPARQL-12 SEP.

Reply:
They can all have reasonable implementation. The groundwork has been laid out. And functions like checking equality, gt, lt, .. are all nice to have.

Reply:
@rubensworks I will just leave this as is (in this PR)?

Reply:
Yep, as-is is fine.